File: Objc/OCTCustomCollectionViewLayout/OCTCustomCollectionViewLayout/OCTCollectionViewCell.m
//
//  OCTCollectionViewCell.m
//  OCTCustomCollectionViewLayout
//
//  Created by fantom on 4/4/17.
//  Copyright © 2017 dmitry.brovkin. All rights reserved.
//

#import "OCTCollectionViewCell.h"

@implementation OCTCollectionViewCell

- (void)awakeFromNib {
    [super awakeFromNib];
    self.imgView.layer.cornerRadius = 8;
    self.imgView.clipsToBounds = YES;
}

@end

File: Swift/OCTCustomCollectionViewLayout/OCTCustomCollectionViewLayout/OCTCollectionViewCell.swift
//
//  OCTCollectionViewCell.swift
//  OCTCustomCollectionViewLayout
//
//  Created by fantom on 4/4/17.
//  Copyright © 2017 dmitry.brovkin. All rights reserved.
//

import UIKit

class OCTCollectionViewCell: UICollectionViewCell {
    @IBOutlet weak var imgView: UIImageView!

    override func awakeFromNib() {
        super.awakeFromNib()
        imgView.layer.cornerRadius = 8
        imgView.clipsToBounds = true
    }

}
